TurkishFilter/sections/general_extensions.txt

```diff
@@ -346,6 +346,8 @@ r10.net#%#document.cookie = 'r10LeftHide=1';
 !------- CSS fixes --------------------!
 !--------------------------------------!
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/82065
+fullhdfilmizlesenebox.com#$#body { background: #000 !important; }
```

Comment:
better:
`||fullhdfilmizlesenebox.com/wp-content/uploads/*/Pageskin.jpg`

Reply:
You are a magician.

Reply:
I have a history of making things complicated.